### PR TITLE
Enhance voting ux

### PR DESCRIPTION
**Pull request by albertkai**

### CODE REVIEW

File: imports/voting/api/store/voting_store.js [hunk 1]
```diff
@@ -38,21 +38,25 @@ export default {
...
       })
     },
-    voteByWeb3({ state }, upVote) {
+    voteByWeb3({ state, commit }, upVote) {
       const { eth: { sign, defaultAccount }, sha3 } = window.web3;
       sign(defaultAccount, sha3(state.proposal.statement), (err, signature) => {
         if (err) return console.error(err);
         Meteor.call('proposals.vote', state.proposal._id, signature, upVote, (err) => {
+          commit('toggleProposalModal');
+          swal({
```
Comment (ducdev) [inline imports/voting/api/store/voting_store.js]: Hey @albertkai I think we shouldn't do anything before checking if there is any error or not ?

File: .meteor/versions [hunk 2]
```diff
@@ -41,6 +41,7 @@ http@1.2.12
 id-map@1.0.9
 jquery@1.11.10
 juliancwirko:postcss@1.2.0
+kevohagan:sweetalert@1.0.0
```
Comment (ducdev) [inline .meteor/versions:44]: We have https://www.npmjs.com/package/vue-sweetalert here.

**Review by ducdev — CHANGES_REQUESTED** [review]
I left comments about sweetalert and callback logic.

File: imports/voting/api/store/voting_store.js (hunk 1 above)
Comment (albertkai) [reply imports/voting/api/store/voting_store.js]: yep, you are right, will fix now)

**Pull request merged**